test/test_base.py

```diff
@@ -0,0 +1,86 @@
+import dataclasses
+
+import numpy as np
+
+from hippopt import (
+    OptimizationObject,
+    Parameter,
+    StorageType,
+    TOptimizationObject,
+    Variable,
+    default_storage_type,
+)
+
+
+@dataclasses.dataclass
+class TestVariable(OptimizationObject):
+    storage: StorageType = default_storage_type(Variable)
+
+
+@dataclasses.dataclass
+class TestParameter(OptimizationObject):
+    storage: StorageType = default_storage_type(Parameter)
+
+
+def test_zero_variable():
+    test_var = TestVariable()
+    test_var.storage = np.ones(shape=3)
+    test_var_zero = test_var.get_default_initialized_object()
+    assert test_var_zero.storage.shape == (3,)
+    assert np.all(test_var_zero.storage == 0)
```

Comment:
```suggestion
    assert test_var_zero.storage == pytest.approx(np.zeros(3))
```

Reply:
in this case, I am expecting the value to be exactly zero. Is this relaxation necessary?

Reply:
Not necessarily, but comparing floats is always tricky. It's ok in this specific case, but at least now you know the `pytest` resources useful to compare Sequences :)

Reply:
> but at least now you know the `pytest` resources useful to compare Sequences :)

Indeed! Thanks for the advice. For the time being, I am not performing any math operation that might result in incorrect comparisons. So I will keep the strict check while I will definitely keep in mind the ``pytest.approx`` utility.